serverless/functions/serverless-functions-on-cluster-builds.adoc

```diff
@@ -0,0 +1,90 @@
+:_content-type: PROCEDURE
+[id="serverless-functions-on-cluster-builds"]
+= Building and deploying functions on-cluster
+:context: serverless-functions-on-cluster-builds
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+You can use the Knative (`kn`) CLI to initiate a function project build directly on the cluster. To build a function project in this way, the source code for your function project must exist in a Git repository branch that is accessible to your cluster.
+
+:FeatureName: {FunctionsProductName}
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[id="prerequisites_serverless-functions-on-cluster-builds"]
+== Prerequisites
+
+* xref:../../cicd/pipelines/installing-pipelines.adoc[{pipelines-title}] must be installed on your cluster.
```

Comment:
+1 to what John says. Technically this is in an assembly now, but this procedure really does need to be moved into its own procedure module

Also, the xref is missing an #anchor

Reply:
Removed the xref anyway when moving to module